docs/aop/user/apps/usercentrics.md

```diff
@@ -0,0 +1,138 @@
+
+{% info_block infoBox "Info" %}
+
+You can configure the data processing services, as well as the visual representation of the cookie consent popups for your store, in the Usercentrics Admin Interface. See the [Usercentrics knowledge base](https://usercentrics.atlassian.net/servicedesk/customer/portals) for details.
+
+{% endinfo_block %}
+
+To integrate Usercentics:
+
+1. In your store's Back Office, go to **Apps->Catalog**.
+2. Click **Usercentrics**.
+   This takes you to the Usercentrics app details page.
+3. In the top right corner of the Usercentrics app details page, click **Connect app**.
+   This displays a message about the successful integration of the app. The Usercentrics app's status changes to *Connection pending*.   
+4. Log in to the [Usercentrics Admin Interface](https://admin.usercentrics.eu/).
+5. In the Usercentrics Admin Interface, copy the setting ID of your app:
+   ![usercentrics-setting-id](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/user/apps/usercentrics/usercentrics-setting-id.png)
+6. Go back to your store's Back Office, to the Usercentrics app details page.
+7. In the top right corner of the Usercentrics app details page, click **Configure**.
+8. In the *Configure* pane, in *Global Settings* by default, *[Enable Smart Data Protector]*(#smart-data-protector) is selected. You can either leave this setting or select *[Enable Direct Integration (Works only with Google Tag manager)]*(#google-tag-manager).
+9. Select the store and insert the setting ID from step 5.
+10. To activate Usercentrics for the selected store, select *Is active*.
+11. Optional: To add more stores with the same or different setting IDs, click *Add store configuration*.
+
+
+That's it. You have integrated the Usercentrics app into your store. The app changes to *Connected* on the apps catalog page. Now, the cookie consent popups should be displayed on every page of the Storefront.
+
+### Global settings configuration
+
+There are three ways to integrate Usecentrics: via direct integration, via the Usercentrics Smart Data Protector, and integration with Google Tag Manager. In the Spryker Back Office, you can select either Smart Data Protector or Google Tag Manager. At the same time, the Smart Data Protector is the preferred and default setting.
+
+{% info_block infoBox "Info" %}
+
+All of the three approaches require you to get the setting ID for your store in the [Usercentrics Admin Interface](https://admin.usercentrics.eu/) and configure the data processing services on page **Service Settings -> Data Processing Services**.
+
+{% endinfo_block %}
+
+#### Direct integration
+
+The direct integration requires mostly manual developer work. You can do the direct integration if you don't use a third-party tracking tool yet.
```

Comment:
We should somehow make it clear, that this is the most basic (and cumbersome way) to do it, and is not supported by AOP and if you customers chose that option they should not use the Spryker UserCentrics integration.